OWNERS_ALIASES

```diff
@@ -45,36 +46,42 @@ aliases:
   release-team-lead-role:
     - guineveresaenger # 1.17 RT Lead
```

Comment:
Last time we had 4 entries including the current lead, are we retaining leads from 4 previous releases in addition to the current lead?

Reply:
@savitharaghunathan I think we keep them as long as a release branch is still active

Reply:
So, we were retaining folks from only the releases that are in-support. Since 1.17 is going to be EOL by end of this month, I am deferring the cleanup to later.

Reply:
Makes sense. Thank you @hasheddan & @palnabarun for the explanation.